### ISSUE TITLE
Update to ember-tether 3.0.0

### ISSUE THREAD

**Issue by arohr** [yapplabs/ember-modal-dialog#394]
Version of ember-tether should be updated here:
https://github.com/yapplabs/ember-modal-dialog/blob/f11dd2fa94a2d2efecf4774bbb8371b40eed46e8/package.json#L86

**Comment by lukemelia** [comment]
This was done in https://github.com/yapplabs/ember-modal-dialog/commit/81178e7dde9d2e9007c9902ffa31d9d20dc619f5 and released as 4.1.1

**Comment by arohr** [comment]
Sure? In package.json I still can see `^2.0.1` here:

```
"peerDependencies": {
    "ember-tether": "^2.0.1"
},
```

**Comment by lukemelia** [comment]
Ah, got it. Missed that one. Thanks!

**Comment by arohr** [comment]
This leads to a npm error breaking my app. Could you please release a fixed version? Thanks!

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR! 
npm ERR! While resolving: ember-modal-dialog@4.1.1
npm ERR! Found: ember-tether@3.0.0
npm ERR! node_modules/ember-tether
npm ERR!   dev ember-tether@"^3.0.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peerOptional ember-tether@"^2.0.1" from ember-modal-dialog@4.1.1
npm ERR! node_modules/ember-modal-dialog
npm ERR!   dev ember-modal-dialog@"^4.1.1" from the root project
npm ERR! 
npm ERR! Conflicting peer dependency: ember-tether@2.0.1
npm ERR! node_modules/ember-tether
npm ERR!   peerOptional ember-tether@"^2.0.1" from ember-modal-dialog@4.1.1
npm ERR!   node_modules/ember-modal-dialog
npm ERR!     dev ember-modal-dialog@"^4.1.1" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.

```

**Comment by lukemelia** [comment]
Will do via #395

**Comment by lukemelia** [comment]
Released in 4.1.2
